config-production.yaml

```diff
@@ -1,8 +1,10 @@
 # Production configuration for the cf-plugin-backup pipelines
 # Note that the secrets are not here; they are in $CONCOURSE_SECRETS_FILE
 
-src-name: SUSE/cf-plugin-backup
-src-repo: git@github.com:SUSE/cf-plugin-backup.git
+src-owner:                   SUSE
+src-repo:                         cf-plugin-backup
```

Comment:
Oh, actually: Since you changed `src-repo`, you need to modify the two other files that refer to it.